suite-native/module-settings/src/screens/SettingsDeviceChecksScreen.tsx

```diff
@@ -1,14 +1,22 @@
-import { useTranslate } from '@suite-native/intl';
+import { Text, VStack } from '@suite-native/atoms';
+import { Translation, useTranslate } from '@suite-native/intl';
 import { Screen, ScreenHeader } from '@suite-native/navigation';
 
+import { TurnOffDeviceAuthenticityCheckCard } from '../components/TurnOffDeviceAuthenticityCheckCard';
 import { TurnOffFirmwareAuthenticityCheckCard } from '../components/TurnOffFirmwareAuthenticityCheckCard';
 
 export const SettingsDeviceChecksScreen = () => {
     const { translate } = useTranslate();
 
     return (
-        <Screen header={<ScreenHeader content={translate('moduleSettings.deviceChecks.title')} />}>
-            <TurnOffFirmwareAuthenticityCheckCard />
+        <Screen header={<ScreenHeader content={translate('moduleSettings.advanced.title')} />}>
```

Comment:
Why not use `Translation` here? 🤔

Reply:
Because if I use Translation here, it has to be also wrapped in Text component here. I also don't like it, but it's consistent with other screens so if we want to refactor it, I would do it as separate PR.

Reply:
OK, I know what you mean. No problem.